cloud/integrations.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Integrate Timescale Cloud services with third-party monitoring tools
+excerpt: Export telemetry metrics to DataDog or AWS CloudWatch
+product: cloud
+keywords: [integrations, DataDog, AWS CloudWatch]
+---
+
+import CloudIntegrationsExporterRegion from 'versionContent/_partials/_cloud-integrations-exporter-region.mdx';
+
+# Integrate Timescale Cloud services with third-party monitoring tools
+
+You can export your service telemetry to a third-party monitoring tool, such as
+[DataDog][datadog] or [AWS CloudWatch][cloudwatch]. To do so, create a data
+exporter and attach your service to it.
+
```

Comment:
I wonder if we need to document this in the form of process/procedure, instead of just procedures?